jobs/integr8ly/ocp3/test-suites/update-sourcecode-test.yaml

```diff
@@ -0,0 +1,88 @@
+---
+
+- job:
+    name: launcher-tests
```

Comment:
I think the name here is incorrect. should be `update-sourcecode-test` if you want to match it with the updated w2-test-executor